### PR TITLE
My portifolio branch

### PR DESCRIPTION
**Pull request by mwakiogeofrey**
This project is about making a mobile portfolio

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1054,13 +1066,14 @@ footer {
...
   flex-grow: 0;
+  margin-left: -70px;
 }
 
 .pic2 {
   width: 18px;
   height: 18px;
   margin-top: 230px;
-  margin-left: 80px;
+  margin-left: -10px;
```
Comment (franclobo) [inline style.css:1076]: - [ ] Kindly use the image in the HTML code and review the path, not all the images are background some of them are part of the content of the page 👇🏽 

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @mwakiogeofrey, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Best practices of HTML & CSS ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- **[OPTIONAL]** Please use a descriptive title and a list of actions in your PR for professional documentation.
- [ ] Please review the width of your page it should be responsive at the media width of 375px but it has a width of 500px that exed the media query for a better user interface the page should be responsive.

https://user-images.githubusercontent.com/58642949/220978239-fdbe589f-c77a-4404-8cf2-85f5a9522ec9.mp4



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -6,72 +6,201 @@
...
+                </div>
+            </div>
+
+        </div>
+       
+            <div class="about">
+                <div>
+                    <h1 class="me">About me</h1>
+                    <p class="hello">Hello I’m a software developer! I can help<br> you build a product , feature or website<br> Look through some of my work and<br> experience! If you like what you see and<br> have a project you need coded, don’t<br> hestiate to contact me.</p>
+                    <h2 class="my">Get My Resume</h2>
```
Comment (Shisui6) [inline index.html:147]: [Optional] Please add some left padding to this button so it better matches the figma design🙏

Your design:
image

Figma design:
image

File: index.html [hunk 3]
```diff
@@ -6,72 +6,201 @@
...
+                        <img class="pic1" src="images/Rectangle 56.png" alt="">
+                        <h1 class="frame">Frameworks</h1>
+                    </div>
+                    <ul class="www">
+                        <li class="B">Bootstrap</li>
+                        <li class="Rr">Ruby on Rails</li>
+                        <li class="P">RSPec</li>
+                        <li class="Cc">CapyBara</li>
+                        <li class="S">Selenium</li>
+                    </ul>
```
Comment (Shisui6) [inline index.html:175]: [Optional] Please make sure that all content in this section is center aligned to better match the figam design🙏

Your design:
image

Figma design:
image

File: index.html [hunk 4]
```diff
@@ -6,72 +6,201 @@
...
+                        <div class="linefive">
+                         <h1 class="seenn">See Project</h1>
+                        </div>
+                </div>
+                <div class="sixs">
+                       <h1 class="eleven">Profesional Art Printing Data</h1>
+                     <h1 class="twelve">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required. has been the<br> industry's standard</h1>
+                      <img src="images/Tags (1).png" alt="">
+                        <div class="linesix">
+                           <h1 class="se">See Project</h1>
```
Comment (Shisui6) [inline index.html:136]: [Optional] Please update this element to be either an `<a>` or `<button>` tag🙏. This is for semantic reasons as this element will be used to perform some dynamic functionality in later projects. Also don't forget to add button interaction styling for when your buttons are hovered and pressed. You can use the `:hover` and `:active` pseudo selectors to style them. Please mak sure to follow the figma style guide for buttons

- Please update the text `see project` to be on the same line

image

**Review by Shisui6 — APPROVED** [review]
Hi @mwakiogeofrey , 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Good design ✅
- Good use of flexbox and grid ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

Please check the comments under the review. These were simply missed by the previous reviewer but are essential to making sure your project is perfect🫡

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**